types/signature_pad/index.d.ts

```diff
@@ -1,137 +1,187 @@
-// Type definitions for signature_pad
+// Type definitions for signature_pad 2.3
 // Project: https://github.com/szimek/signature_pad
 // Definitions by: Abubaker Bashir <https://github.com/AbubakerB>
 //                 Jason Mihalick <https://github.com/jrmihalick>
+//                 Jaeyeon Lee <https://github.com/leejaedus>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // Project by: Szymon Nowak <https://github.com/szimek>
+// TypeScript Version: 2.8
 
-declare namespace SignaturePad {
-	class Point {
-	    x: number;
-	    y: number;
-		time: number;
-		color?: string;
-	    constructor(x: number, y: number, time: number);
-	    velocityFrom(start: Point): number;
-	    distanceTo(start: Point): number;
+declare module 'signature_pad' {
```

Comment:
This is not the 2.x way of making a UMD module. The easiest way is actually to keep the old structure with `declare namespace SignaturePad` and then at the end add:

```ts
declare namespace SignaturePad {
  // ... original structure ...
}
export default SignaturePad
export as namespace SignaturePad
```

The new `export as namespace` syntax implements UMD by creating a global namespace named SignaturePad, but only in non-module-using programs.